apps/src/templates/instructions/codeReviewV2/Comment.jsx

```diff
@@ -110,8 +110,18 @@ class Comment extends Component {
     );
   };
 
+  deleteCodeReviewComment = () => {
+    this.props.onDelete(
+      this.props.comment.id,
+      () => {},
+      () => {
+        // TODO: Handle comment delete failure
```

Comment:
is there a follow-up task for this?

Reply:
I kept this TODO here since I saw a similar TODO for handing resolve state failure, and I think there are plans to refactor this component a bit - @maureensturgeon is there a follow-up task for addressing the TODOs in `V2/Comment`? I can also handle the failure case in this PR if that's easier.

Reply:
Yeah, I've been displaying a generic CodeReviewError in some other places. You could do that here. It will probably be worth it to do a pass and create more specific error messages for all the error cases. If you can handle those error cases, that'd be great!

Reply:
Sounds good, for now I can keep it similar to V1 where we flash an error message on error.